packages/object-utilities/src/js/__tests__/__protected-test.js

```diff
@@ -0,0 +1,42 @@
+jest.dontMock("../__protected");
+
+// import { __protected } from "../__protected";
+
+const __protected = require("../__protected").default;
+
+describe("ProtectedUtil", () => {
+  it("throw error without object", () => {
+    expect(() => {
+      __protected();
+    }).toThrowError();
+  });
+
+  it("call without value creates empty protected property and returns that object", () => {
```

Comment:
Please adjust the test description according to the guidelines http://www.betterspecs.org/